FrascatiPlugin.php

```diff
@@ -45,10 +45,35 @@ public function register($category, $path, $mainContextId = null)
         if ($success && $this->getEnabled()) {
             // Add hook
             Hook::add('API::vocabs::external', $this->setData(...));
+            Hook::add('Form::config::after', $this->addVocabularyToKeywordsField(...));
+
         }
         return $success;
     }
 
+     public function addVocabularyToKeywordsField($hookName, $args) {
+
+        if ($args[0]["id"] == 'metadata') {
+            // Find the keywords field
+            foreach ($args[0]["fields"] as $key => $field) {
+                if ($field["name"] == "keywords") {
+                    // Add the vocabulary field
+                    $jsonPath = dirname(__FILE__) . '/classifications.en.json';
+                    $args[0]["fields"][$key]["vocabularies"] = [
+                        [
+                            "locale" => "en",
+                            "addButtonLabel" => "Add Frascati Keywords",
+                            "title" => "Add Keywords from Frascati Taxonomy",
```

Comment:
Needs localization -- `en` shouldn't be hard-coded, the English text should come from a locale file, and the `.en.` in the JSON filename shouldn't be assumed.

Reply:
Right, just to confirm. I think we agreed having the json for individual languages as separate files, right? So you are referencing here just the button label and title, right? But keeping the vocabulary items in the json, and not putting them to the locale files.

Reply:
Yes, keeping the JSON in separate files for each languages is best, thanks -- and make sure to use `en` as a fallback when a file isn't present. I'd like eventually to offload the translations elsewhere, like with [CRediT](https://github.com/contributorshipcollaboration/credit-translation), but doubtful we'll get update on that in the short term!